### ISSUE TITLE
Training stucked in 'plotting labels'

### ISSUE THREAD

**Issue by xiuluoNM** [ultralytics/yolov5#5611]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello，I'm very interested in yolov5, and I have trained several models before, but yesterday I meet a problem: when I'm training my custom data, it has stucked in cmd print 'plotting labels', and I tried everything but it can't continue. Following is the cmd 

lines:
Transferred 364/370 items from yolov5s.pt
Scaled weight_decay = 0.0005
optimizer: SGD with parameter groups 59 weight, 70 weight (no decay), 62 bias
train: Scanning '..\smoke2\train\labels' images and labels...303 found, 0 missing, 0 empty, 0 corrupted: 100%|█| 303/30
train: New cache created: ..\smoke2\train\labels.cache
val: Scanning '..\smoke2\valid\labels' images and labels...6 found, 0 missing, 0 empty, 0 corrupted: 100%|█| 6/6 [00:03
val: New cache created: ..\smoke2\valid\labels.cache
Plotting labels...

(ff) D:\test\Yolov5_DeepSort_Pytorch-master-pig\yolov5>

I want to know why ....

**Comment by glenn-jocher** [comment]
@xiuluoNM might be a possible duplicate of https://github.com/ultralytics/yolov5/issues/5395

The conclusion there was that this was caused by an issue in `freetype`. Downgrading `freetype` to 2.10.4 fixed the issue.

**Comment by glenn-jocher** [comment]
@xiuluoNM good news 😃! Your original issue may now be fixed ✅ in PR #5616. This PR does not fix any underlying issues with matplotlib/freetype, but it does enclose plot_labels() in `try: except` and `Timeout` decorators to bypass it in case of issues. This means no label plots will be produced if errors/hangs are encountered, but training will proceed normally without issue.

https://github.com/ultralytics/yolov5/blob/def7a0fd19c1629903c3b073b4df265407719a07/utils/plots.py#L327-L331

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
